pkg/images/images_test.go

```diff
@@ -194,8 +194,8 @@ func Test_releaseToKeep(t *testing.T) {
 		{
 			name: "min < 2.7.0 < 2.9.0 < max",
```

Comment:
need to be updated to 
```suggestion
			name: "min < 2.8.0 < 2.9.0 < max",
```